components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/ApplicationDAO.java

```diff
@@ -124,10 +124,11 @@ default String getApplicationResourceIDByInboundKey(String inboundKey, String in
      * Get authenticators configured for an application.
      *
      * @param applicationId ID of an application.
+     * @param tenantDomain
      * @return Authentication configurations.
      * @throws IdentityApplicationManagementException
      */
-    LocalAndOutboundAuthenticationConfig getConfiguredAuthenticators(String applicationId)
+    LocalAndOutboundAuthenticationConfig getConfiguredAuthenticators(String applicationId, String tenantDomain)
```

Comment:
Here also consider having method overloading. If needed the existing method can be deprecated to use the new method.